openedx/features/course_experience/api/v1/views.py

```diff
@@ -73,6 +77,19 @@ def reset_course_deadlines(request):
         if missed_deadlines and not missed_gated_content:
             reset_self_paced_schedule(user, course_key)
 
+            course_overview = course_detail(request, user.username, course_key)
+            # For context here, research_event_data should already contain `location` indicating
+            # the page/location dates were reset from and could also contain `block_id` if reset
+            # within courseware.
+            research_event_data.update({
+                'courserun_key': str(course_key),
+                'is_masquerading': is_masquerading(user, course_key, course_masquerade),
```

Comment:
This is basically `bool(course_masquerade)` but I guess if we have a utility method, ought to use it.

Reply:
Yeah, my same thoughts. If the underlying logic ever changes at some point, good to stay in line with it.